backend/apps/dashboard/tests.py

```
@@ -164,3 +55,17 @@ def test_cursor_identical_xp(self):
         # However, user0, user1, user10, user11... will be the alphabetical order
         sorted_usernames = sorted(usernames)
         self.assertEqual(usernames, sorted_usernames)
+
+class IssueModelTests(APITestCase):
+    def setUp(self):
+        self.user = User.objects.create_user(username="testuser", password="password")
```

Ruff (0.15.20)

[error] Possible hardcoded password assigned to argument: "password"

(S106)